Libraries/Microsoft.Teams.Common/Http/HttpClient.cs

```diff
@@ -73,37 +73,35 @@ protected HttpRequestMessage CreateRequest(IHttpRequest request)
 
         Options.Apply(httpRequest);
 
-        foreach (var kv in request.Headers)
-        {
-            if (kv.Key.StartsWith("Content-"))
-            {
-                httpRequest.Content?.Headers.TryAddWithoutValidation(kv.Key, kv.Value);
-                continue;
-            }
-
-            httpRequest.Headers.TryAddWithoutValidation(kv.Key, kv.Value);
-        }
-
         if (request.Body is not null)
         {
             if (request.Body is string stringBody)
             {
                 httpRequest.Content = new StringContent(stringBody);
-                return httpRequest;
             }
-
-            if (request.Body is IEnumerable<KeyValuePair<string, string>> dictionaryBody)
+            else if (request.Body is IEnumerable<KeyValuePair<string, string>> dictionaryBody)
             {
                 httpRequest.Content = new FormUrlEncodedContent(dictionaryBody);
-                return httpRequest;
             }
-
-            httpRequest.Content = JsonContent.Create(request.Body, options: new JsonSerializerOptions()
+            else
             {
-                DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingNull
-            });
+                httpRequest.Content = JsonContent.Create(request.Body, options: new JsonSerializerOptions()
+                {
+                    DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingNull
+                });
+            }
         }
 
+        foreach (var kv in request.Headers)
```

Comment:
Alex, I need to put code for setting Content header after setting up Httprequest.Content from body, without this "Content-" header were not set.

Reply:
Makes sense because `Content` would be overridden in the previous if statement